Dockerfile

```diff
@@ -53,7 +53,23 @@ RUN set -ex \
     && ls -lah /app/
 
 # Runtime container
-FROM base
+FROM base as app
 USER app:app
 WORKDIR /app
 COPY --from=build --chown=root:root /app /app
+
+# Load testing container
+FROM base as loadtest
```

Comment:
you trying to owerride the default DOckerfile build. You need to create another one Dockerfile or include all your packages inside app

Reply:
Ok, Ill just remove this